kong/tools/queue.lua

```diff
@@ -100,6 +100,97 @@ local function make_queue_key(name)
 end
 
 
+local function internal_remaining_capacity(self)
+  local remaining_entries = self.max_entries - self:count()
+
+  -- we enqueue entries one by one,
+  -- so it is impossible to have a negative value
+  assert(remaining_entries >= 0, "queue should not be over capacity")
+
+  if self.max_bytes == nil then
```

Comment:
Why not use `not self.max_bytes`?

Reply:
There is no particular reason, just personal habits.

Reply:
See: https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#conditional-expressions,
I think `not var` may be better.

Reply:
fixed